build.solr.xml

```diff
@@ -195,9 +195,9 @@ ${line.separator}$config['search_api.server.solr_unpublished']['backend_config']
     </target>
 
     <target name="purge-solr-backend" description="Empty Solr cores">
```

Comment:
While doing `install-dev`, I get:
```
My Drupal Project > purge-virtuoso-backend:

sh: /usr/bin/isql-vt: not found
sh: write error: Broken pipe
```

Reply:
Fixed by providing a new target upstream to clear the triples using http as well.